test/test_attention.py

```diff
@@ -80,24 +85,31 @@ def generate_none_mask(batch, seq_length, dtype=torch.float32):
 )
 @pytest.mark.parametrize("implementation", implementations.keys())
 def test_benchmark_masked(
-    benchmark, shape: (int, int), implementation: Callable, mask_fn: Callable, dtype: torch.dtype, is_causal: bool
+    benchmark,
+    shape: (int, int, int, int),
+    implementation: Callable,
+    mask_fn: Callable,
+    dtype: torch.dtype,
+    is_causal: bool,
 ):
-    batch, seq_length = shape
-    if implementation == "original" and (dtype == torch.bfloat16 or seq_length != 512):
-        pytest.skip("Original Triton implementation only supports fp16 and seq_length=512")
-    elif implementation == "original" and mask_fn != generate_none_mask:
-        pytest.skip("Original Triton implementation doesn't support masks")
-
-    # batch, heads, seq_length, dhead
-    mat_shape = (batch, 48, seq_length, 64)
+    batch, heads, seq_length, dhead = shape
+    if implementation == "original":
+        if dtype == torch.bfloat16 or seq_length != 512:
+            pytest.skip("Original Triton implementation only supports fp16 and seq_length=512")
+        elif mask_fn != generate_none_mask:
+            pytest.skip("Original Triton implementation doesn't support masks")
+
+    # bf16 reduced precision is sensitive to this value
+    scale = 1.0 if dtype == torch.bfloat16 else 2.0
```

Comment:
could we rename this ? it will create confusions for sure. It's the max value of input tensors in reality